spec/features/homepage/homepage-carousel.automater.js

```diff
@@ -0,0 +1,107 @@
+const webdriver = require('selenium-webdriver');
+
+const By = webdriver.By;
+
+
+class Automater {
+	constructor(options) {
+		const handleError = (msg) => {
+			console.error(msg); // eslint-disable-line no-console
+		};
+
+		this.driver = options.driver || handleError('');
+		this.carouselItemSelector = options.carouselItemSelector || handleError('');
+		this.carouselActiveItemImageSelector = options.carouselActiveItemImageSelector || handleError('');
+		this.carouselSelector = options.carouselSelector || handleError('');
+		this.searchControlSelector = options.searchControlSelector || handleError('');
+	}
+
+	checkForRotation(rotationInterval) {
+		const self = this;
+		return new Promise((resolve, reject) => self.getActiveImgSrc(self)
+			.then((src) => {
+				setTimeout(() => self.getActiveImgSrc(self)
+					.then((newSrc) => {
+						const isRotating = src !== newSrc;
+						resolve(isRotating);
+					}), rotationInterval);
+			})
+			.catch(reject));
+	}
+
+	// eslint-disable-next-line class-methods-use-this
+	getActiveImgSrc(automater) {
+		return new Promise((resolve, reject) =>
+			automater.driver.findElement(By.css(automater.carouselActiveItemImageSelector))
+				.then(elm => elm.getAttribute('src')
+					.then(resolve)
+					.catch(reject))
+			.catch(reject));
+	}
+
+	static getCssValueByElement(elm, prop) {
```

Comment:
To keep things organized, would you keep your static methods together if possible?